src/lms-ds-loader/edfi_lms_ds_loader/loader_facade.py

```diff
@@ -40,7 +39,6 @@ def run_loader(arguments: MainArguments) -> None:
     db_adapter = arguments.get_db_operations_adapter()
 
     _load_users(csv_path, db_adapter)
```

Comment:
Similar to what we do when extracting data, it could be helpful to handle exceptions here, so in case a non-critical resource load fails, you can continue with the others and log an error message without breaking the load.

Reply:
That's an interesting idea. 

Currently, let's say that there's a failure when sections loads. The users are already loaded at that point - and we don't wrap a transaction around the entire process, so the users won't roll back.

Let's say that system activities comes after sections in this class. What you're proposing is that we go ahead and load the system activities even though the sections failed. This seems consistent with the fact that we already loaded the users. So I like it. Great suggestion!

Reply:
I ended up writing some unit tests on the `loader_facade` so that I could prove that the try/catch is behaving as expected (which of course is obvious when you look at it, but now the tests also serve as documentation that this is all on purpose). What you suggested about why I was having trouble with monkey patching was true, so you'll see some changes to the import statements on the `loader_facade.py` file.

To test this, I ran the application on the google output. Then changed the name of the `LMSSection` table temporarily in my database, and ran again. Confirmed that I get the desired final logging of the exception, the final warning message, and that the exit code is 1 instead of 0.

![image](https://user-images.githubusercontent.com/9324390/110704977-53d0ea80-81bb-11eb-8819-8f3985768cd0.png)

Reply:
😎